src/utils/preparedPeople.tsx

```diff
@@ -0,0 +1,39 @@
+import { Person } from '../types';
+
+export const preparedPeople = (
+  searchParams: URLSearchParams,
+  people: Person[],
+) => {
+  const query = searchParams.get('query') || null;
+  const centuries = searchParams.getAll('centuries') || null;
+  const sex = searchParams.get('sex') || null;
+
+  let copyPeople = [...people];
```

Comment:
The `filter` method returns a copy of the array, so you don't need to do it manually.

Reply:
> The `filter` method returns a copy of the array, so you don't need to do it manually.

Agree